src/components/submission/feed-entry.vue

```diff
@@ -31,6 +31,17 @@ except according to the terms contained in the LICENSE file.
             <template #name><actor-link :actor="entry.actor"/></template>
           </i18n-t>
         </template>
+        <template v-else-if="entry.action === 'entity.create'">
+          <span class="icon-check entity-icon submission-feed-entry-icon"></span>
+          <i18n-t keypath="title.entity.create">
+            <template #label><span class="submission-feed-entry-entity-data">{{ entityLabel(entry) }}</span></template>
+            <template #dataset><span class="submission-feed-entry-entity-data">{{ entityDataset(entry) }}</span></template>
+          </i18n-t>
+        </template>
+        <template v-else-if="entry.action === 'entity.create.error'">
+          <span class="icon-warning entity-icon submission-feed-entry-icon"></span>
+          <span class="submission-feed-entry-entity-error" :title="entityProblem(entry)">{{ $t('title.entity.error') }}</span>
```

Comment:
It might be nice to make the `entry.details.problem.problemDetails.reason` more visible at some point, maybe by putting it in the `.body` below. But I think putting it in the `title` is a good place to start.

Reply:
Linking to this backend issue about what to do when submission-to-entity processing fails: https://github.com/getodk/central-backend/issues/661